### DISCUSSION TITLE
another idea for better lod/chunk control

### DISCUSSION THREAD

**Discussion by octanejohn** [TokisanGames/Terrain3D#556] [Questions]
it started with unity and now intel by using concurrent binary tree for terrain/ocean or even for regular mesh objects

https://arxiv.org/pdf/2407.02215
https://m.youtube.com/watch?v=-Ce-XC4MtWk

**Comment by TokisanGames** [comment]
Thanks. One of our devs looked at it and it isn't really relevant for how we're creating meshes.
